hoprd/rest-api/src/node.rs

```diff
@@ -40,8 +43,8 @@ pub(crate) struct NodeVersionResponse {
     )]
 pub(super) async fn version(State(state): State<Arc<InternalState>>) -> impl IntoResponse {
     let version = state.hopr.version();
-
-    (StatusCode::OK, Json(NodeVersionResponse { version })).into_response()
+    let api_version = env!("CARGO_PKG_VERSION").to_owned();
```

Comment:
This is not needed at all. The `openapi.json` schema contains the version number and is always available with in the API.

Reply:
See the discussion under the issue.